src/main/java/EOorg/EOeolang/EOsys/EOcall.java

```diff
@@ -136,45 +105,25 @@ public EOcall(final Phi sigma) {
      * @param rho The \rho
      * @return ID
      */
-    private static int number(final Phi rho) {
+    private static SysCall sysfunc(final Phi rho) {
         final String txt = new Param(rho, "id").strong(String.class);
-        final Integer cid;
-        if (txt.matches("[0-9]+")) {
-            cid = Integer.parseInt(txt);
-        } else {
-            cid = EOcall.GLOSSARY.get(txt);
-        }
-        if (EOcall.GLOSSARY.isEmpty()) {
-            throw new ExFailure(
-                String.format(
-                    "It's impossible to syscall at this OS: '%s'",
-                    EOcall.UNAME
-                )
-            );
-        }
-        if (cid == null) {
+        if (!Glossary.contains(txt)) {
```

Comment:
@mximp I think it's more logical to put this validator into `Glossary.syscall` method and get rid of `contains()`

Reply:
fixed